### PR TITLE
Check for dbus-1 when building with cloudproviders

### PR DESCRIPTION
**Pull request by nextch**
Resolves: #4153
Signed-off-by: nextch <97217716+nextch@users.noreply.github.com>

### CODE REVIEW

**Review by allexzander — APPROVED** [review]
Looks sane to me

File: CMakeLists.txt [hunk 1]
```diff
@@ -182,6 +182,7 @@ if(BUILD_CLIENT)
       pkg_check_modules(CLOUDPROVIDERS cloudproviders IMPORTED_TARGET)
 
       if(CLOUDPROVIDERS_FOUND)
+        pkg_check_modules(DBUS-1 REQUIRED dbus-1 IMPORTED_TARGET)
```
Comment (mgallien) [inline CMakeLists.txt:185]: sorry for the late review
I do not really get why we check for something we will not use `dbus-1` is never used
we do not use it when linking and we never use it to locate include headers

so my next question is why do you want that check ?

Comment (nextch) [reply CMakeLists.txt:185]: No worries!

The presence of `dbus-1` seems to be expected in [shell_integration/libcloudproviders/CMakeLists.txt](https://github.com/nextcloud/desktop/blob/40021e69438846b24e7c801dbca604aefbaf5a79/shell_integration/libcloudproviders/CMakeLists.txt#L22-L28). If you have `cloudproviders` but don't have `dbus-1`, then you get a build error that doesn't say what exactly is missing and you have to dig through the build files in order to figure it out. See also #4153.

Comment (mgallien) [reply CMakeLists.txt:185]: sorry missed this access to dbus-1 package
so thanks a lot for the fix !!!

**Review by mgallien — APPROVED** [review]
thanks !!!!

**Comment by mgallien** [comment]
@nextch can you rebase ?
we need that to merge your PR

**Comment by mgallien** [comment]
> @nextch can you rebase ? we need that to merge your PR

sorry to ask you again but another PR was merged before yours and I need again you to rebase
sorry for the inconvenience

**Pull request merged**